go/proto/ctrl/v1/acme.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package ctrl.v1;
+
+option go_package = "github.com/unkeyed/unkey/go/gen/proto/ctrl/v1;ctrlv1";
+
+import "google/protobuf/timestamp.proto";
+
+message HandleCertificateVerificationRequest {
```

Comment:
I'm open for a differen name

Reply:
it definitely shouldn't be `HandleXxxx` cause that sounds like a function.
rabbits suggestion looks pretty good, wdyt?

Reply:
@coderabbitai please create an issue for us to rename this proto definition